x/deployment/types/event.go

```diff
@@ -34,8 +45,19 @@ func (ev EventDeploymentCreate) ToSDKEvent() sdk.Event {
 
 // EventDeploymentUpdate struct
 type EventDeploymentUpdate struct {
-	ID      DeploymentID
-	Version []byte // TODO
+	Context sdkutil.BaseModuleEvent `json:"context"`
+	ID      DeploymentID            `json:"id"`
+	Version []byte                  `json:"version,omitempty"` // TODO
```

Comment:
should we mention issue for every place version needs fix?

Reply:
probably, yeah; good call.